editor/project_manager.cpp

```diff
@@ -376,20 +376,33 @@ class ProjectDialog : public ConfirmationDialog {
 		fdialog_install->popup_file_dialog();
 	}
 
-	void _create_folder() {
-		const String project_name_no_edges = project_name->get_text().strip_edges();
+	void _create_new_project_folder() {
+		_create_folder(project_name->get_text(), project_path);
+	}
+
+	void _create_install_folder() {
+		Vector<String> strings = project_path->get_text().split("/");
+		String filename = strings[strings.size() - 1];
+		String filename_no_extension = filename.split(".")[0];
```

Comment:
If the file name has multiple periods (e.g. `project.sample.zip`), this means the file name would be considered to be `project`, which is incorrect.

Instead, you can use `filename.rstrip(filename.get_extension())`.

Reply:
Fixed, I'll test and then amend the commit.